### ISSUE TITLE
Support for Developer and "Nightly" editions of Firefox

### ISSUE THREAD

**Issue by klabarge** [qzind/tray#217]
In addition to the stable version of Firefox, there are "Firefox Developer Edition" and "Firefox Nightly" versions: https://www.mozilla.org/en-US/firefox/channel/desktop/

QZ Tray does not work over wss with these versions. Since these versions are not stable, and unlikely to rolled out to production users, is it worth adding support for these versions?

One pro, is that we will be able to fully test QZ Tray + SSL before the features even hit the beta release.

**Comment by tresf** [comment]
Should be similar to the trick we use for ESR releases, at least on Windows.

https://github.com/qzind/tray/blob/2.0/ant/windows/windows-keygen.js.in#L376

**Comment by tresf** [comment]
Firefox Developer Edition:
```diff
- var firefoxKey = "HKLM\\Software\\Mozilla\\Mozilla Firefox";
+ var firefoxKey = "HKLM\\Software\\Mozilla\\Firefox Developer Edition";
```

Firefox Nightly:
```diff
- var firefoxKey = "HKLM\\Software\\Mozilla\\Mozilla Firefox";
+ var firefoxKey = "HKLM\\Software\\Mozilla\\Nightly";
```

We'll need to determine how best to shim this into the installer (if at all).

Unlike ESR, Firefox seems to encourage having this installed side-by-side Firefox-stable ,so we may want to install to all versions, which complicates the logic.

**Comment by tresf** [comment]
Closed via #504.

**Comment by tresf** [comment]
As @sheck points out, it looks like we missed this for MacOS, reopening.
